### DISCUSSION TITLE
Hot to change the favicon of the status page itself

### DISCUSSION THREAD

**Discussion by mulyoved** [upptime/upptime#153] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/2309907/103440428-02056480-4c4e-11eb-905f-d354cc1c3ac3.png)

**Comment by AnandChowdhary** [comment]
This is currently not supported. I'll make sure I push and update to add support for custom icons within this weekend!

**Reply by ririko5834** [reply]
When this will be added?

**Reply by AnandChowdhary** [reply]
Soon, I promise! I'll try and find some time in the next few days for sure. Sorry for the delay!

**Reply by ririko5834** [reply]
@AnandChowdhary Can you please also add support for custom notifications? 
That you can specify custom request url, headers, method, body, ...

**Reply by FxllenCode** [reply]
> Soon, I promise! I'll try and find some time in the next few days for sure. Sorry for the delay!

Any updates for this? I would love to see this, especially because the Favicon doesn't work _at all_ at this point:

![mXrlKnqPj7](https://user-images.githubusercontent.com/40151425/105429944-79605f80-5c20-11eb-99f0-097ce91ad877.png)


This would be really appreciated!

**Comment by tomershvueli** [comment]
+1 would love to see this!

**Comment by wolfier74** [comment]
Good idea please add it soon

**Comment by AnandChowdhary** [comment, marked as answer]
You can change the favicon by setting these properties:

```yaml
status-page:
  faviconSvg: https://example.com/favicon.svg
  favicon: https://example.com/favicon.png
```
